crates/ggml/src/tensor.rs

```diff
@@ -15,6 +15,36 @@ impl Tensor {
     /// Exposed for purposes of determining context size.
     pub const C_TYPE_SIZE: usize = std::mem::size_of::<sys::ggml_tensor>();
 
+    ///Sets the name of the tensor
+    pub fn set_name(&mut self, name: &str) -> &Tensor {
+        assert!(name.len() <= 48, "Name is too long!");
+
+        let bytes = name.as_bytes();
+        let mut array = [0i8; 48];
+        array[..bytes.len()].copy_from_slice(&bytes.iter().map(|&x| x as i8).collect::<Vec<_>>());
+
+        unsafe { self.ptr.as_mut().name = array }
+        self
+    }
+
+    ///Gets the name of the tensor
+    pub fn get_name(&self) -> String {
```

Comment:
just `name`, ditto with `get_backend` - Rust house style appears to be to use function names that are the same as the variable for getters